renovate.json

```diff
@@ -1,31 +1,26 @@
 {
-  "enabled": true,
-  "semanticCommits": true,
-  "rebaseStalePrs": true,
-  "labels": [
-    "renovate"
-  ],
+  "extends": ["config:js-app", ":semanticCommits", ":rebaseStalePrs"],
+  "labels": ["renovate"],
   "packageRules": [
     {
       "packageNames": [
-        "@dcos/http-service",
-        "@dcos/mesos-client",
-        "@dcos/tslint-config",
-        "@dcos/ui-kit",
-        "@dcos/data-service",
-        "@extension-kid/core",
-        "typescript",
-        "react",
-        "react-dom",
-        "react-test-renderer"
+        "browser-info",
+        "clipboard",
+        "d3",
+        "flux",
+        "graphql-tools",
+        "inversify",
+        "query-string",
+        "react-ace",
+        "react-bootstrap-typeahead",
+        "react-router",
+        "redux"
```

Comment:
i never used renovate before. do i get correctly that it'll only create PRs for the packages declared in `packageNames`? if so, how is this list compiled?

Reply:
no, this config applies special settings to these packages - I added them manually. Its a list of things that are behind in major version (we're using redux v3, but most recent is v5)

Idea is to manually update these packages and remove them from this list.

Reply:
everything thats not in this list will be updated to `latest` by renovate.

Reply:
now i also understand the things in the trade-offs-section. thanks!